src/lib/stitching/kaws/__tests__/stitching.test.ts

```diff
@@ -1,6 +1,26 @@
 import { runQuery } from "test/utils"
 
-describe("MarketingCollectionArtwork", () => {
+// TODO: The tests in this file make _actual_ requests to KAWS.
+xdescribe("MarketingCollectionArtwork", () => {
```

Comment:
Lol, just realizing these tests should still be pended from https://github.com/artsy/metaphysics/pull/1512 , they were subsequently un-pended....they make actual http requests.